README.md

```diff
@@ -1,146 +1,25 @@
 # Overview
 
+![tcl-42smartshoppinglist](https://user-images.githubusercontent.com/86992220/170784008-9494285a-d273-4203-8f3d-ad233c6ae778.png)
+
 ## What is this project?
 
-The goal of this project is to collaborate with your Collab Lab team to create a “smart” shopping list app that learns your buying habits and helps you remember what you’re likely to need to buy on your next trip to the store.
+This is a smart shopping app developed by a group of three developers under [The Collab Lab](https://the-collab-lab.codes/about-us/) mentorship program. This sleek app allows users to keep track of the items they purchase and learns the purchase frequency those items. This app is mobile and accessibility (a11y) friendly. 
 
 ## How does it work?
+As a user, you can create a shopping list which is associated with a three word token. You can also use a shared token to access a shopping list. Once inside your list, you can add/delete items to/from your shopping list. Each time you buy an item the app will estimate and log the item's next purchase date. The list is sorted by purchase frequency of the items with color styles and accessibility labels for the different tiers of purchase frequencies (soon, kind-of-soon, not-soon, inactive). Available on the list is a search bar for searching through your shopping list for specific items.
```

Comment:
Add 'existing' to the following sentence: You can also use a shared token to access an **existing** (delete shopping) list. 
Delete 'shopping' out of the same^ sentence.

Reply:
Add 'Also', 'page', and 'filter' to the following sentence: **Also** available on the list **page** is a search bar for searching**/filtering** through your shopping list for specific items.